Source/WebCore/css/html.css

```diff
@@ -1341,6 +1341,10 @@ html:focus, body:focus, input[readonly]:focus, embed:focus, iframe:focus, object
     outline: none;
 }
 
+input[readonly] {
+    cursor: text;
+}
```

Comment:
We already have rules for input and textarea here: https://searchfox.org/wubkat/rev/eb60a003e2487c2a8f34aaf8ce8f5d5af280ed55/Source/WebCore/css/html.css#476
https://searchfox.org/wubkat/rev/eb60a003e2487c2a8f34aaf8ce8f5d5af280ed55/Source/WebCore/css/html.css#797

We should either change that rule or change whatever auto computes to.

Reply:
The rule you added (wrongly) applies to all input types, not all input types support readonly.

Reply:
I would try to change https://searchfox.org/wubkat/rev/eb60a003e2487c2a8f34aaf8ce8f5d5af280ed55/Source/WebCore/page/EventHandler.cpp#1606